frontend/src/components/OdhListItem.scss

```diff
@@ -1,41 +1,41 @@
 .odh-list-item {
   &.m-disabled {
-    background-color: var(--pf-global--BackgroundColor--light-200);
+    background-color: var(--pf-v5-global--BackgroundColor--light-200);
     position: relative;
     &:before {
       position: absolute;
       top: 0;
       right: 0;
       left: 0;
-      height: var(--pf-c-card--m-selectable--m-selected--before--Height);
-      content: "";
-      background-color: var(--pf-global--disabled-color--200);
+      height: var(--pf-v5-c-card--m-selectable--m-selected--before--Height);
+      content: '';
+      background-color: var(--pf-v5-global--disabled-color--200);
     }
   }
   &__doc-text {
-    color: var(--pf-global--Color--100);
-    font-size: var(--pf-global--FontSize--sm);
+    color: var(--pf-v5-global--Color--100);
+    font-size: var(--pf-v5-global--FontSize--sm);
     overflow: hidden;
     text-overflow: ellipsis;
     white-space: nowrap;
   }
   &__doc-title {
-    color: var(--pf-global--Color--100);
+    color: var(--pf-v5-global--Color--100);
     overflow: hidden;
     text-overflow: ellipsis;
     white-space: nowrap;
   }
   &__doc-description {
-    color: var(--pf-global--Color--200);
+    color: var(--pf-v5-global--Color--200);
     overflow: hidden;
     text-overflow: ellipsis;
     white-space: nowrap;
   }
   &__link {
-    font-size: var(--pf-global--FontSize--sm) !important;
+    font-size: var(--pf-v5-global--FontSize--sm) !important;
     > svg {
-      font-size: var(--pf-global--FontSize--xs) !important;
-      margin-left: var(--pf-global--spacer--xs);
+      font-size: var(--pf-v5-global--FontSize--xs) !important;
+      margin-left: var(--pf-v5-global--spacer--xs);
```

Comment:
The icon structure is changed in PF5, the `svg` is wrapped by 2 `span` elements for the icons, which causes the style cannot apply and breaks the spacer. This happens on `Resources` page when you switch to a listview. See the space between `Read how-to article` and the `ExternalLinkAltIcon`

<img width="264" alt="Screenshot 2023-11-01 at 2 29 48 PM" src="https://github.com/opendatahub-io/odh-dashboard/assets/37624318/236db694-4886-44d3-9a46-9045bb7edc7b">
<img width="884" alt="Screenshot 2023-11-01 at 2 31 05 PM" src="https://github.com/opendatahub-io/odh-dashboard/assets/37624318/65d3ba59-160e-4004-9e73-41351e3d5551">

Reply:
Thank you @DaoDaoNoCode  fixed the spacing by updating the component [to use an inline link button with icon](https://github.com/opendatahub-io/odh-dashboard/pull/2179/files#diff-3bf3406e62079f431d2cf5b382360a7a0baab6970f6af9a157a43b0956051f56R72)